docs/source/tutorials/resnet/index.md

```diff
@@ -12,48 +12,170 @@ LightlyTrain can be installed directly from PyPI:
 pip install lightly-train
 ```
 
-## Dataset Preparation
+## Pretrain ResNet with LightlyTrain
+
+### Download the Pretraining Dataset
+
+Here we use [a dataset for human detection on Kaggle](https://www.kaggle.com/datasets/killa92/human-detection-dataset) for pretraining. It contains 170 PNG images (of which 146 for training and 24 for validation) of various sizes containing humans.
+
+You can download the dataset directly from Kaggle using the following commands (suppose you want the dataset to locate in `datasets`):
+
+```bash
+curl -L -o datasets/human-detection-dataset-pretraining.zip \
+https://www.kaggle.com/api/v1/datasets/download/killa92/human-detection-dataset
```

Comment:
This command doesn't work on linux because `curl` doesn't create the `datasets` directory if it doesn't exist. There are two options:
* Add the `--create-dirs` flag to let curl create the directory
* Do not save the dataset into an extra `datasets` directory